installer/mac/SymphonySettingsPlugin/SymphonySettingsPlugin/MyInstallerPane.m

```diff
@@ -14,21 +14,40 @@ - (NSString *)title
     return [[NSBundle bundleForClass:[self class]] localizedStringForKey:@"PaneTitle" value:nil table:nil];
 }
 
-- (void)willExitPane:(InstallerSectionDirection)dir {
-    
-    // Set the default protocol to https
-    NSString *protocol = @"https://";
+- (void)willEnterPane:(InstallerSectionDirection)dir {
+    // By default, set the value of the error message textbox to an empty string
+    [_podUrlAlertTextBox setTitleWithMnemonic:@""];
+}
+
+- (BOOL)shouldExitPane:(InstallerSectionDirection)dir {
     
     NSString *podUrl = [_podUrlTextBox stringValue];
     
-    // If the pod url is empty, by default, set it to my.symphony.com
-    if ([podUrl length] == 0) {
-        podUrl = @"my.symphony.com";
+    // Check if the url contains a protocol, if not, prepend https to it
+    NSString *prefix = @"https://";
+    if (![podUrl hasPrefix:prefix]) {
+        podUrl = [prefix stringByAppendingString:podUrl];
+        [_podUrlTextBox setStringValue:podUrl];
+    }
+    
+    // Now, validate the url against a url regex
+    NSString *regex = @"^((?:http:\/\/)|(?:https:\/\/))(www.)?((?:[a-zA-Z0-9]+\.[a-z]{3})|(?:\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}(?::\d+)?))([\/a-zA-Z0-9\.]*)$";
```

Comment:
Having only "www" might be a problem here. What if PODs are run on subdomains?

Reply:
`www` is an optional (`?` indicates optional). This works for sub-domains too. Please check https://regex101.com/ with the above regex, both sub-domains and full domains work with or without `www`.